loader.s

```diff
@@ -10,7 +18,7 @@ align 4
   dd CHKSUM
 
 loader:
-  mov eax, 0xdeadbeef
```

Comment:
This line is for testing whether the bootloader worked, and `0xdeadbeef` was just a joke.